GOVERNANCE.md

```diff
@@ -50,42 +46,23 @@ and so on.
 
-The current committers are (in addition to the TC members above):
+The current committers are (along with their primary areas(s) of ownership or maintenance):
 
-* [@hugs](https://github.com/hugs)
-* [@filmaj](https://github.com/filmaj)
-* [@aluedeke](https://github.com/aluedeke)
-* [@sebv](https://github.com/sebv)
-* [@vgrigoruk](https://github.com/vgrigoruk)
-* [@DylanLacey](https://github.com/DylanLacey)
-* [@maudineormsby](https://github.com/maudineormsby)
-* [@titusfortner](https://github.com/titusfortner)
-* [@bayandin](https://github.com/bayandin)
-* [@jdeff](https://github.com/jdeff)
-* [@moizjv](https://github.com/moizjv)
-* [@gempesaw](https://github.com/gempesaw)
-* [@Jonahss](https://github.com/Jonahss)
-* [@JamieEdge](https://github.com/JamieEdge)
-* [@saikrishna321](https://github.com/saikrishna321)
-* [@sravanmedarapu](https://github.com/sravanmedarapu)
-* [@peepa](https://github.com/peepa)
-* [@scottdixon](https://github.com/scottdixon)
-* [@drpy](https://github.com/drpy)
-* [@Astro03](https://github.com/Astro03)
-* [@sbonebrake](https://github.com/sbonebrake)
-* [@SrinivasanTarget](https://github.com/SrinivasanTarget)
-* [@triager](https://github.com/triager)
-* [@ddkjin](https://github.com/ddkjin)
-* [@stuartbrussell-intuit](https://github.com/stuartbrussell-intuit)
-* [@vmaxim](https://github.com/vmaxim)
-* [@Dan-Maor](https://github.com/Dan-Maor)
-* [@laolubenson](https://github.com/laolubenson)
-* [@Dor-bl](https://github.com/Dor-bl)
-* [@boneskull](https://github.com/boneskull)
+* [@Dan-Maor](https://github.com/Dan-Maor) - XCUITest driver and iOS internals
+* [@Dor-bl](https://github.com/Dor-bl) - DotNet client
+* [@KazuCocoa](https://github.com/KazuCocoa) - all Appium projects
+* [@SrinivasanTarget](https://github.com/SrinivasanTarget) - Java client
+* [@eglitise](https://github.com/eglitise) - Appium server, Inspector
+* [@jlipps](https://github.com/jlipps) - Appium server, Inspector
+* [@laolubenson](https://github.com/laolubenson) - DotNet client
 * [@mwakizaka](https://github.com/mwakizaka)
+* [@mykola-mokhnach](https://github.com/mykola-mokhnach) - all Appium projects
 * [@rerorero](https://github.com/rerorero)
+* [@saikrishna321](https://github.com/saikrishna321) - Java client
 * [@tomriddly](https://github.com/tomriddly)
```

Comment:
mwakizaka, rerorero and tomriddly are listed both here and in the Emeritus section, is that okay?